lib/src/rules/member_ordering.dart

```diff
@@ -236,12 +283,24 @@ class _Annotation {
       _Annotation._('HostBinding', _MembersGroup.angularHostBindings);
   static const hostListener =
       _Annotation._('HostListener', _MembersGroup.angularHostListeners);
+  static const viewChild =
+      _Annotation._('ViewChild', _MembersGroup.angularViewChild);
+  static const viewChildren =
+      _Annotation._('ViewChildren', _MembersGroup.angularViewChild);
+  static const contentChild =
+      _Annotation._('ContentChild', _MembersGroup.angularContentChild);
+  static const contentChildren =
+      _Annotation._('ContentChildren', _MembersGroup.angularContentChild);
 
   static List<_Annotation> annotations = [
```

Comment:
@incendial can we make private this variable ?

Reply:
Fixed